ci/ciimage/arch/install.sh

```diff
@@ -15,6 +15,7 @@ pkgs=(
   doxygen vulkan-validation-layers openssh mercurial gtk-sharp-2 qt5-tools
   libwmf cmake netcdf-fortran openmpi nasm gnustep-base gettext
   python-lxml hotdoc rust-bindgen qt6-base qt6-tools wayland wayland-protocols
+  debugedit glib2 glib2-devel
```

Comment:
Disable debug support in AUR packages, instead. We are not e.g. testing that valgrind can perform valgrind on scalapack / wxwidgets-gtk2 so there's no reason to spend longer compiling and produce bigger docker containers.